app/models/task.rb

```diff
@@ -27,9 +27,11 @@ class Task < ApplicationRecord
   ranks :row_order, with_same: :project_id
 
   belongs_to :project, required: true
+  has_many :comments, as: :commentable, dependent: :destroy
```

Comment:
could you pls move these commentable stuff into a concern and just include in any model needed